src/test/groovy/com/github/vkorobkov/jfixtures/sql/DataSourceUtil.groovy

```diff
@@ -0,0 +1,24 @@
+package com.github.vkorobkov.jfixtures.sql
+
+import org.apache.commons.dbcp2.BasicDataSource
+import spock.lang.Shared
+
+import javax.sql.DataSource
+
+class DataSourceUtil {
+    private static final String DATASOURCE_URL = "jdbc:h2:mem:jfixturesDb;DB_CLOSE_DELAY=-1;DATABASE_TO_UPPER=false"
+
+    private static DataSource dataSource
+
+    @Shared
```

Comment:
Hm, not really and can be removed.. But @Shared is more for readers to show them that this variable is the same for all feature methods.. making tests more readable.